e2e/eslint/src/linter.test.ts

```diff
@@ -447,6 +446,17 @@ describe('Linter', () => {
           ];
           return json;
         });
+        process.env.NX_E2E_EDITOR = 'code';
+      });
+
+      afterAll(() => {
```

Comment:
Ensure dependency checks are cleared out after this test to not pollute the rest of the suite.